PalasoUIWindowsForms/Progress/LogBox.cs

```diff
@@ -287,7 +287,7 @@ private void Write(Color color, FontStyle style, string msg, params object[] arg
 						rtfBoxForDelegate.SelectionColor = color;
 						rtfBoxForDelegate.SelectionFont = fnt;
 #endif
-						rtfBoxForDelegate.AppendText(string.Format(msg + Environment.NewLine, args));
+						rtfBoxForDelegate.AppendText(GenericProgress.SafeFormat(msg + Environment.NewLine, args));
```

Comment:
I agree.  When I pick this up again, I could gladly remove the GenericProgress method entirely (with appropriate fanfare on the email list)